src/main/java/com/feedzai/commons/sql/abstraction/engine/DatabaseEngineDriver.java

```diff
@@ -38,6 +38,20 @@ public String connectionStringFor(String hostname, String database, int port) {
             return connectionStringFor(hostname, database);
         }
     },
+    /**
+     * The H2v2 Database vendor.
+     */
+    H2V2("H2V2", "com.feedzai.commons.sql.abstraction.engine.impl.H2EngineV2", "org.h2.Driver", 0) {
+        @Override
+        public String connectionStringFor(String hostname, String database) {
+            return String.format("jdbc:h2:%s/%s", hostname, database);
+        }
+
+        @Override
+        public String connectionStringFor(String hostname, String database, int port) {
```

Comment:
We don't care about the port?

Reply:
I just followed the convention that was already being used by `H2`. I guess that's because this code relates to the embedded mode of H2, which does not use a port.